src/tribler-core/tribler_core/modules/metadata_store/community/remote_query_community.py

```diff
@@ -148,20 +154,30 @@ async def process_rpc_query(self, json_bytes: bytes):
         request_sanitized = sanitize_query(json.loads(json_bytes), self.settings.max_response_size)
         return await self.mds.get_entries_threaded(**request_sanitized)
 
-    def send_db_results(self, peer, request_payload_id, db_results):
+    def send_db_results(self, peer, request_payload_id, db_results, force_eva_response=False):
         index = 0
         while index < len(db_results):
-            data, index = entries_to_chunk(db_results, self.settings.maximum_payload_size, start_index=index)
+            transfer_size = (
+                self.eva_protocol.binary_size_limit if force_eva_response else self.settings.maximum_payload_size
+            )
```

Comment:
**nit** probably it is not necessary to use `(` and `)` here.

Reply:
this style of using braces is forced by `black` for the reason it is safer to use `()` instead of `\` for breaking ternary stuff

Reply:
:)